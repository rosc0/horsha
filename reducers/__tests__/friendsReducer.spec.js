import reducer, { initialState as state } from '../friends'
// import types from '../../constants/actionTypes'

// import { horse_picture, horse_friends } from '../../../test/api'

describe('Comments Reducer', () => {
  it('should return initial state', () => {
    expect(reducer()).toBe(state)
    expect(reducer()).toMatchSnapshot()
  })
})
