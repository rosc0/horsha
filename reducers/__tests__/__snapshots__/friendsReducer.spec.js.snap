// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Reducer should return initial state 1`] = `
Object {
  "error": null,
  "fetched": false,
  "fetching": false,
  "friendSearch": Array [],
  "friendSearchRemaining": 0,
  "friends": Array [],
  "friendsByKm": Array [],
  "friendsByKmCursor": null,
  "friendsByKmRemaining": 0,
  "friendsCursor": null,
  "friendsRemaining": 0,
}
`;
